src/js/closable-surveys.js

```diff
@@ -30,6 +30,7 @@ import LocalStorageManager from './localStorageManager';
       try {
         surveysToHide = JSON.parse(window.localStorage.getItem('helfi-settings'));
       } catch (e) {
+        // eslint-disable-next-line no-console
```

Comment:
Should we tweak the rules to allow these? I think we're using `console.error` and capturing these to Sentry